lib/api/core/entrypoints/embedded/protocols/http.js

```diff
@@ -22,23 +22,37 @@
 'use strict';
 
 const
+  assert = require('assert'),
   debug = require('../../../../../kuzzleDebug')('kuzzle:entry-point:protocols:http'),
   assert = require('assert'),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Correct this attempt to modify "assert" or use "let" in its declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3500)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "assert" as this name is already used in declaration at line 25. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS2814)